subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <stdint.h>
+#include <zephyr.h>
+#include <flash.h>
+#include <download_client.h>
+#include <nrf_socket.h>
```

Comment:
Do you need headers like `nrf_socket.h` or `bsd.h`?

Reply:
No, removed.